### DISCUSSION TITLE
Need help understand tp.file.cursor(2)

### DISCUSSION THREAD

**Discussion by thesamim** [SilentVoid13/Templater#866] [Help] [answered]
I'm probably missing something fundamental, but I can get the second cursor position to be parsed or used.

`<% tp.file.cursor(1) %> ` works fine, and positions the cursor there on "Open today's daily  note".
but
`It should go here: <% tp.file.cursor(2) %> ` doesn't seem to be parsed/translated. It just renders as text in my note.

Would appreciate any pointers to what I'm missing.

Thanks in advance.

Here is my  daily notes template:
`
```
---
creation date: <% tp.file.creation_date() %>
modification date: <% tp.file.last_modified_date("dddd Do MMMM YYYY HH:mm:ss") %>
---
# <% tp.file.title %>
```
---
tags: daily-note
full-date: <% tp.file.title %>
week: <% tp.date.now("YYYY-[W]ww", 0, tp.file.title, "YYYY-MM-DD") %>
month: <% tp.date.now("YYYY - MM-MMMM", 0, tp.file.title, "YYYY-MM-DD") %>
year: <% tp.date.now("YYYY", 0, tp.file.title, "YYYY-MM-DD") %>
---
<< [[<% tp.date.now("YYYY-MM-DD", -1) %>]] | [[<% tp.date.now("YYYY-MM-DD", 1) %>]] >>
# My Day
## Journal
<% tp.file.cursor(1) %>
## Todoist
```todoist
 { "name": "My Tasks", "filter": "today | overdue | no due date" } 
```
## Other tasks
```tasks
due before [[<% tp.date.now("YYYY-MM-DD", 1) %>]]
not done
```
## Random Stuff
It should go here: <% tp.file.cursor(2) %>
`

**Comment by AB1908** [comment, marked as answer]
Are you looking for the command palette command `Jump to next cursor`?

**Reply by thesamim** [reply]
Actually, I was aware of that hotkey. But what I was not aware of was that the text  (`<% tp.file.cursor(2) %>`) doesn't get replaced until the jump key is invoked. Which I just now discovered by invoking the jump key...

Thanks for the pointer!
